man/systemd-udevd.service.xml

```diff
@@ -105,6 +105,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-s=</option></term>
+        <term><option>--timeout-signal=</option></term>
+        <listitem>
+          <para>Set the signal which systemd-udevd will send to the forked off
+          process after reaching event timeout.</para>
```

Comment:
`to the forked off process`→ `to forked off processes`, i.e. it will it apply to any processes, i.e. likely more than once, hence use plural